service/internal/db/db.go

```diff
@@ -183,3 +187,18 @@ func (d SQLClient) ListTokenEntries(ctx context.Context, project string) ([]Toke
 	err = sess.WithContext(ctx).Collection(TokenEntryDB).Find("project", project).OrderBy("-created_at").All(&res)
 	return res, err
 }
+
+func optionsToMap(options string) map[string]string {
+	opts := map[string]string{}
+	options = strings.TrimSpace(options)
+
+	if options != "" {
+		kvPairs := strings.Split(options, " ")
+		for _, entry := range kvPairs {
+			kv := strings.Split(entry, "=")
```

Comment:
I typically use [`SplitN`](https://pkg.go.dev/strings#SplitN) for these cases as it will provide the upper bound of items back (in the case there is a `=` in the _value_). The doc for `SplitN` says to use [`Cut`](https://pkg.go.dev/strings#Cut) for this case (which also handles if there is no `=` at all (as it will return `false`).